lib/Server.js

```diff
@@ -94,9 +94,10 @@ class Server {
     this.contentBaseWatchers = [];
 
     // TODO this.<property> is deprecated (remove them in next major release.) in favor this.options.<property>
-    this.hot = this.options.hot || this.options.hotOnly;
-    this.headers = this.options.headers;
-    this.progress = this.options.progress;
+    this.hot = options.hot || options.hotOnly;
+    this.liveReload = options.liveReload;
```

Comment:
Don't use `this.liveReload` (this variables will be removed in next major release), please use `this.options.liveReload` in code